lib/streamlit/DeltaGenerator.py

```diff
@@ -1299,7 +1299,7 @@ def image(
         )
 
     @_with_element
-    def audio(self, element, data, format="audio/wav"):
+    def audio(self, element, data, current_time=0, format="audio/wav"):
```

Comment:
Can you update the API above and the docs below to match the latest one I just posted in the bug? This one:

```
st.audio(data, format="audio/wav", start_time=0)
st.video(data, format="video/mp4", start_time=0)
```

Same with `video`.

---
Sorry about this change! This API wasn't finalized yet when you implemented it. Next time we'll be more clear about the status of a proposed API in our feature requests.